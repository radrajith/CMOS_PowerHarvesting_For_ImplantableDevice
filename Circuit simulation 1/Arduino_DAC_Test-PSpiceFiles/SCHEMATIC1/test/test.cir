** Profile: "SCHEMATIC1-test"  [ C:\USERS\RADRA\BOX SYNC\SENIOR FALL 2016\ESE 440\Circuit simulation 1\Arduino_DAC_Test-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\radra\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 100ms 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
