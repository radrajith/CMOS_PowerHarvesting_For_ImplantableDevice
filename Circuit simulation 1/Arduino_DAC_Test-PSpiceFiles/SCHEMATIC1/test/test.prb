[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG MANUAL
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
REFMIN 0.0291497099088606
REFMAX 0.0375086939256327
REFUNIT s
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG MANUAL
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
REFMIN 1.62249964757823
REFMAX 1.65214460101962
REFUNIT V
BEGIN TRACE V(V1:+)
MARKERID 2
TRACEADDEXT
END TRACE V(V1:+)
BEGIN TRACE V(C1:2)
MARKERID 3
TRACEADDEXT
END TRACE V(C1:2)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
