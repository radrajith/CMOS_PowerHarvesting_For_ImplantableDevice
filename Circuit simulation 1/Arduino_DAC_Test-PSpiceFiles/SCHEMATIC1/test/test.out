
**** 11/09/16 21:32:20 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\RADRA\BOX SYNC\SENIOR FALL 2016\ESE 440\Circuit simulation 1\Arduino_DAC_Test-PSpiceFiles\


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\radra\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 100ms 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source ARDUINO_DAC_TEST
R_R1         N00612 N00067  10k TC=0,0 
C_C1         0 N00067  50n  TC=0,0 
V_V1         N00158 0  
+PULSE 0 5 1ns 1ns 1ns 0.0000192 0.000032
M_M1         N00158 N00733 N00612 N00612 MbreakN           
V_V2         N00733 0  
+PULSE 0 5 1ns 1ns 1ns 50ms 100ms
C_C2         0 N00067  10p  TC=0,0 

**** RESUMING test.cir ****
.END

**** 11/09/16 21:32:20 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\RADRA\BOX SYNC\SENIOR FALL 2016\ESE 440\Circuit simulation 1\Arduino_DAC_Test-PSpiceFiles\


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               MbreakN         
               NMOS            
       LEVEL    1            
           L  100.000000E-06 
           W  100.000000E-06 
         VTO    0            
          KP   20.000000E-06 
       GAMMA    0            
         PHI     .6          
      LAMBDA    0            
          IS   10.000000E-15 
          JS    0            
          PB     .8          
        PBSW     .8          
          CJ    0            
        CJSW    0            
        CGSO    0            
        CGDO    0            
        CGBO    0            
         TOX    0            
          XJ    0            
       UCRIT   10.000000E+03 
      DIOMOD    1            
         VFB    0            
        LETA    0            
        WETA    0            
          U0    0            
        TEMP    0            
         VDD    5            
       XPART    0            


**** 11/09/16 21:32:20 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\RADRA\BOX SYNC\SENIOR FALL 2016\ESE 440\Circuit simulation 1\Arduino_DAC_Test-PSpiceFiles\


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N00067) 286.7E-21 (N00158)    0.0000 (N00612) 286.7E-21 (N00733)    0.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1         0.000E+00
    V_V2         0.000E+00

    TOTAL POWER DISSIPATION   0.00E+00  WATTS



          JOB CONCLUDED

**** 11/09/16 21:32:20 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\RADRA\BOX SYNC\SENIOR FALL 2016\ESE 440\Circuit simulation 1\Arduino_DAC_Test-PSpiceFiles\


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =        1.24
